history-of-the-quakers/original/28-book-3-chapter-2.adoc

```diff
@@ -347,21 +351,24 @@ and published the rest in their justification to the nation at large.
 Abstract of a declaration from the people called Quakers, against all sedition, plotters,
 fighters, etc. presented to the king the 21st of 11th month, 1660.
 
-"`Our principle is, and our practice hath always been,
+[.embedded-content-document.address]
```

Comment:
Not sure if this type should be blank....I thought an address was the closest match.

Reply:
Works for me. 👍